src/components/SearchBar.js

```diff
@@ -283,6 +288,18 @@ class SearchBar extends React.Component<SearchBarDefaultProps, SearchBarProps, S
     }
   }
 
+  //eslint-disable-next-line
+  shareSearch() {
+    const username = AuthUtils.getUserName(AuthUtils.getSavedUser());
+    const yourMessage =
+      'Hey,\n\nI think you would be interested in these search results that I found using the most cognitive and ' +
```

Comment:
This message should be a property. In a PoV this makes sense, but if someone wants to use it in a production system, they'd probably want to have different text.